reflex/vars/base.py

```diff
@@ -1559,8 +1559,9 @@ def __init__(
         Raises:
             TypeError: If the computed var dependencies are not Var instances or var names.
         """
-        hints = get_type_hints(fget)
-        hint = hints.get("return", Any)
+        hint = kwargs.pop("return_type", None) or get_type_hints(fget).get(
```

Comment:
should we put `return_type` in the actual `__init__` signature? seems a little hard to find/undocumented this way

Reply:
it might be intentionally that way (using might + intentionally in something i did is a bit questionable, but alas), i did it because users should almost always pass the return type through the function return signature, sometimes in reflex code we don't have that option so it's more of an internal thing.